runner/src/internal.rs

```diff
@@ -249,38 +296,81 @@ impl Runnable for Cmd {
                 }
             }
 
-            let output = child
-                .wait_with_output()
-                .expect("could not wait for command to terminate");
+            Running {
+                command: command_string,
+                process: Some(child),
+            }
+        }
+    }
+}
+
+/// An instance of a running command.
+pub struct Running {
+    command: String,
+    process: Option<std::process::Child>,
+}
 
-            let mut logs = String::new();
-            {
-                let stdout =
-                    std::str::from_utf8(&output.stdout).expect("could not parse command stdout");
-                if !stdout.is_empty() {
-                    logs += &format!("════╡ stdout ╞════\n{}", stdout);
+impl Running {
+    /// Forces the running command to stop. Equivalent to sending `SIGKILL`.
```

Comment:
The shell scripts are carefully set up to start by sending `SIGINT` (to which the Runtime responds with a graceful shutdown) and only sending `SIGKILL` if the `SIGINT` didn't work.  I think we will need the same here &ndash; and even better if we could treat the need for a `SIGKILL` as an indicator that the test failed.  (A TODO as a `Future<EquivalentFunctionality>` is fine though).